pkg/runtime/nodes.go

```diff
@@ -123,6 +133,10 @@ func (k *KubeadmRuntime) deleteNodes(nodes []string) error {
 			if err := k.deleteNode(node); err != nil {
 				return fmt.Errorf("delete node %s failed %v", node, err)
 			}
+			err := k.deleteVIPRoute(node)
```

Comment:
Why delete route here?

Reply:
> Why delete route here?

after deleting a node, clear the traces of route addition.